src/pybind/mgr/orchestrator.py

```diff
@@ -878,8 +878,8 @@ def list_daemons(self, daemon_type=None, daemon_id=None, host=None, refresh=Fals
         """
         raise NotImplementedError()
 
-    def remove_daemons(self, names):
-        # type: (List[str]) -> Completion
+    def remove_daemons(self, names, force):
```

Comment:
```suggestion
    def remove_daemons(self, names, force=False):
```

But we can do this in a different PR.